README.md

```diff
@@ -15,6 +15,16 @@ This information reflects the head of this branch.
 | ------------------------------------------------------------------------------------------ | ---------------------------------------| --------------- |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-provisioner:v1.0.1  | 1.13            |
 
+## Feature status
+
+Various external-provisioner releases come with different alpha / beta features. Alpha features are disabled by default and must be explicitly enabled on command line. Beta features are enabled by default and can be disabled on command line. Check `--help` output for alpha/beta features in each release.
+
+Following table reflects the head of this branch.
+
+| Feature  | Status | Description                                                                                |
+| -------- | ------ | ------------------------------------------------------------------------------------------ |
+| Topology | Beta   | [Topology aware dynamic provisioning](https://kubernetes-csi.github.io/docs/topology.html) |
```

Comment:
And restore from snapshot? It's not feature gated, should it be?

Reply:
@xing-yang, do we want to have feature gate for snapshot restore?

IMO, someone installed shapshotter and its CRDs, it could be enough to automatically enable feature in the provisioner.

Reply:
We should at least still say it's alpha

Reply:
Added snapshots as alpha feature, with a note that it cannot be turned off.